MH4U Dex/RegionItemTableViewController.m

```diff
@@ -0,0 +1,87 @@
+//
+//  RegionItemTableViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/6/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "RegionItemTableViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "CoreDataController.h"
+#import "Area.h"
+#import "AreaDrop.h"
+#import "Item.h"
+
+#import "AreaDropTableViewCell.h"
+#import "ItemContainerViewController.h"
+
+@interface RegionItemTableViewController ()
+
+@property (nonatomic, strong) NSArray *areaArray;
+@property (nonatomic, strong) NSMutableDictionary *areaDrops;
+
+@end
+
+@implementation RegionItemTableViewController
+
+- (void)viewDidLoad
+{
+    CoreDataController *coreDataController = [CoreDataController sharedCDController];
+    self.areaDrops = [NSMutableDictionary dictionary];
+    NSError *fetchError = nil;
+    NSSortDescriptor *areaSortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"name" ascending:YES];
+    NSFetchRequest *areaFetchRequest = [[NSFetchRequest alloc] initWithEntityName:[Area entityName]];
+    NSPredicate *areaPredicate = [NSPredicate predicateWithFormat:@"%K == %@", @"region", self.region];
+    //TODO: Support all ranks.
+    NSPredicate *rankPredicate = [NSPredicate predicateWithFormat:@"%K == %@", @"rank", @"Low"];
```

Comment:
Could you use mogenerator-provided constants instead of the string literal here and two lines up and two more lines up?